docs/sources/datasources/google-cloud-monitoring/_index.md

```diff
@@ -8,27 +8,19 @@ weight = 200
 
 # Using Google Cloud Monitoring in Grafana
```

Comment:
Is now a good time to update this title to the new format per your template, or do you want to wait until we do an actual rewrite of the whole thing?

Reply:
I am hoping the developer's/SME's will work on the actual re-write and give us some feedback on how the template. I will create a separate PR to change the titles of all the data source topics.